packages/app/src/elements/history-entry-dialog.ts

```diff
@@ -0,0 +1,121 @@
+import { translate as $l } from "@padloc/locale/src/translate";
+import { ItemHistory, Field } from "@padloc/core/src/item";
+import { confirm } from "../lib/dialog";
+import { Dialog } from "./dialog";
+import { html, css } from "lit";
+import { customElement } from "lit/decorators.js";
+import { repeat } from "lit/directives/repeat.js";
+
+@customElement("pl-history-entry-dialog")
+export class HistoryEntryDialog extends Dialog<ItemHistory, boolean> {
+    private _historyEntry: ItemHistory;
+
+    static styles = [
+        ...Dialog.styles,
+        css`
+            :host {
+                --pl-dialog-max-width: 40em;
+            }
+        `,
+    ];
+
+    // TODO: show vault changes
+
+    renderContent() {
+        if (!this._historyEntry) {
+            return html``;
+        }
+
+        const { fieldsChanged, date } = this._historyEntry;
+
+        return html`
+            <div class="padded vertical spacing layout">
+                <h1 class="big margined text-centering">${$l("History Entry")}</h1>
+                <h2 class="margined text-centering">${$l("Created at {0}", date as unknown as string)}</h2>
+
+                <div class="stretch"></div>
+
+                ${fieldsChanged.includes("name") ? this._showChangedNameContent() : null}
+                ${fieldsChanged.includes("tags") ? this._showChangedTagsContent() : null}
+                ${fieldsChanged.includes("fields") ? this._showChangedFieldsContent() : null}
+
+                <div class="horizontal evenly stretching spacing layout top-margined">
+                    <pl-button class="primary" @click=${() => this._restore()}>
+                        ${$l("Restore item to this version")}
+                    </pl-button>
+                    <pl-button @click=${this.dismiss}> ${$l("Cancel")} </pl-button>
+                </div>
+            </div>
+        `;
+    }
+
+    async show(historyEntry: ItemHistory) {
+        await this.updateComplete;
+        this._historyEntry = historyEntry;
+        return super.show();
+    }
+
+    private _showChangedNameContent() {
+        return html`
+            <div class="tags border-bottom">
+                <h2 class="subtle horizontally-double-margined bottom-margined animated section-header">
+                    <pl-icon icon="text" class="inline small light"></pl-icon>
+                    ${$l("Name")}
+                </h2>
+                <div class="bottom-margined">${this._historyEntry.name || $l("<None>")}</div>
+            </div>
+        `;
+    }
+
+    private _showChangedTagsContent() {
+        return html`
+            <div class="tags border-bottom">
```

Comment:
`tags` seems to be the wrong class for this. Better use `horizontal layout`.

Reply:
Actually, better to display tags below the header, same as in the item view.

Reply:
I think this way makes it more obvious that's what changed (related to the above).

Reply:
Not sure what you mean? The `tags` class is meant to be used explicitly as a container for tags (`<div class="tag"></div>` ) and is simply out of place here. Or are you talking about horizontal vs vertical layout?

Reply:
I changed the class. Was referring mostly to the 

> Actually, better to display tags below the header, same as in the item view.

comment.